wandb/cli/cli.py

```diff
@@ -1618,5 +1619,12 @@ def verify(host):
             "Signed URL requests not made over https. SSL is required for secure communications.",
         )
     wandb_verify.check_wandb_version(api)
-    wandb_verify.check_run(api)
-    wandb_verify.check_artifacts()
+    check_run_success = wandb_verify.check_run(api)
+    check_artifacts_success = wandb_verify.check_artifacts()
```

Comment:
Looking at all these conditions, you might consider instantiating a WandbVerify object instead.
I think that would end up being much cleaner:
```
verify = wandb_verify.Verify()
verify.check()
if verify.failed:
    sys.exit(1)
```
And verify check:
```
def check(self):
    self.check_run(self._api)
    self.check_artifacts()
```
Then you can log failure info in your object and it will read better

(this feedback is for a future PR, i think this is good for now)